use std::error::Error;

pub fn serve() -> Result<(), Box<dyn Error>> {
    // TODO: start backend
    Ok(())
}
